docs/CHANGES.rst

```diff
@@ -0,0 +1 @@
+../CHANGES.rst
```

Comment:
Is it possible to have the header here and then link in the rest of the content from the CHANGES file?

I think I consciously used .txt instead of .rst because, along with LICENSE, I viewed the changelog as a file that should open in anyones default text edit when they unpack a tarball and have a minimum of formatting noise.

That may be a flawed thought, but it was a thought I had.